testing/server-tracetesting/features/http_test/08_rerun_http_test.yml

```diff
@@ -22,9 +22,6 @@ spec:
   - selector: span[name = "query INSERT"]
     assertions:
     - attr:tracetest.selected_spans.count = 1
-  - selector: span[name = "exec UPDATE"]
-    assertions:
-    - attr:tracetest.selected_spans.count = 1
```

Comment:
What is the rationale for removing this assertion? Don't we need to update the test anymore?

Reply:
Good question. This was an RunUpdate made after creating the new run when rerunning a test, to update the state. I removed it incorrectly. Readding it